### PR TITLE
Packaging creation for SentinelOne

### PR DESCRIPTION
**Pull request by v-spadarthi**
Change(s):

See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution SentinelOne.

See guidance below
Version Updated:
2.0.0

See guidance below
Testing Completed:
The link for test results is
https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B46FBF256-CB02-49B8-B258-504E20485047%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20SentinelOne.docx&action=default&mobileredirect=true&share=IQFW8vtGAsu4SbJYUE4gSFBHARV3fLNKael6MbkgIbBYY-A

See guidance below
This is an 3P solution.

### CODE REVIEW

File: Solutions/SentinelOne/Data/Solution_SentinelOne.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "Name": "SentinelOne",
+    "Author": "Microsoft",
```
Comment (v-atulyadav) [inline Solutions/SentinelOne/Data/Solution_SentinelOne.json:3]: Hey @v-spadarthi, Please add Author email

**Review by v-atulyadav — COMMENTED** [review]
Hey @v-spadarthi, please change data connector text as per text document link is broken
image

**Review by v-atulyadav — COMMENTED** [review]
Hi @v-spadarthi, please change text from `Azure Sentinel` to `Microsoft Sentinel`.

File: Solutions/SentinelOne/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-sentinelone",
+	"firstPublishDate": "2022-04-01",
+	"providers": [ "Microsoft " ],
```
Comment (v-atulyadav) [inline Solutions/SentinelOne/SolutionMetadata.json:5]: Hey @v-spadarthi, please remove extra suffix space from provider value.
image

**Comment by v-spadarthi** [comment]
@v-atulyadav :Review comments addressed.

**Review by v-atulyadav — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**